src/main/java/org/jasig/portlet/announcements/mvc/servlet/AjaxApproveController.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Licensed to Apereo under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work
+ * for additional information regarding copyright ownership.
+ * Apereo licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain a
+ * copy of the License at the following location:
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.jasig.portlet.announcements.mvc.servlet;
+
+import java.util.Date;
+
+import javax.portlet.PortletException;
+import javax.servlet.http.HttpServletRequest;
+
+import org.jasig.portlet.announcements.model.Announcement;
+import org.jasig.portlet.announcements.service.IAnnouncementService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.cache.ehcache.EhCacheCacheManager;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.ModelAndView;
+
+/** @author eolsson */
```

Comment:
:art: left over `@author` tag

Reply:
I'll remove it, though the question of how we feel about `@author` tags in the Announcements portlet is still somewhat open.  (Every module or project is entitled to its own culture.)